docs/interpreter/jdbc.md

```diff
@@ -0,0 +1,224 @@
+
+```
+# Add jdbc connector jar
+ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/jdbc/jars/RedshiftJDBC41-1.1.10.1010.jar"
+ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/jdbc/jars/mysql-connector-java-5.1.6.jar"
+```
+You can add all the jars you need to make multiple connections into the same interpreter. To create the interpreter you must specify the parameters, for example  we will create two connections to PostgreSQL and Redshift, the respective prefixes are `default` and `redshift`:
+
+<table class="table-configuration">
+  <tr>
+    <th>name</th>
+    <th>value</th>
+  </tr>
+  <tr>
+    <td>common.max_count</td>
+    <td>1000</td>
+  </tr>
+  <tr>
+    <td>default.driver</td>
+    <td>org.postgresql.Driver</td>
+  </tr>
+  <tr>
+    <td>default.password</td>
+    <td>`********`</td>
+  </tr>
+  <tr>
+    <td>default.url</td>
+    <td>jdbc:postgresql://localhost:5432/</td>
+  </tr>
+  <tr>
+    <td>default.user</td>
+    <td>gpadmin</td>
+  </tr>
+  <tr>
+    <td>redshift.driver</td>
+    <td>com.amazon.redshift.jdbc4.Driver</td>
+  </tr>
+  <tr>
+    <td>redshift.password</td>
+    <td>`********`</td>
+  </tr>
+  <tr>
+    <td>redshift.url</td>
+    <td>jdbc:redshift://examplecluster.abc123xyz789.us-west-2.redshift.amazonaws.com:5439</td>
+  </tr>
+  <tr>
+    <td>redshift.user</td>
+    <td>redshift-user</td>
+  </tr>      
+</table>
+
+
+### Bind to Notebook
+In the `Notebook` click on the `settings` icon at the top-right corner. Use select/deselect to specify the interpreters to be used in the `Notebook`.
+
+### More Properties
+You can modify the interpreter configuration in the `Interpreter` section. The most common properties are as follows, but you can specify other properties that need to be connected.
+
+ <table class="table-configuration">
+   <tr>
+     <th>Property Name</th>
+     <th>Description</th>
+   </tr>
+   <tr>
+     <td>{prefix}.url</td>
+     <td>JDBC URL to connect, the URL must include the name of the database </td>
+   </tr>
+   <tr>
+     <td>{prefix}.user</td>
+     <td>JDBC user name</td>
+   </tr>
+   <tr>
+     <td>{prefix}.password</td>
+     <td>JDBC password</td>
+   </tr>
+   <tr>
+     <td>{prefix}.driver</td>
+     <td>JDBC driver name.</td>
+   </tr>
+   <tr>
+     <td>common.max_result</td>
+     <td>Max number of SQL result to display to prevent the browser overload. This is  common properties for all connections</td>
+   </tr>      
+ </table>
+
+To develop this functionality use this [method](http://docs.oracle.com/javase/7/docs/api/java/sql/DriverManager.html#getConnection(java.lang.String,%20java.util.Properties)). For example if a connection needs a schema parameter, it would have to add the property as follows:
```

Comment:
this is probably due to markdown parsing - is there a different url we could use here?